amt/migrations/versions/69243fd24222_create_user_table_with_uuid_as_pk.py

```diff
@@ -0,0 +1,41 @@
+"""create user table with uuid as pk
+
+Revision ID: 69243fd24222
+Revises: 22298f3aac77
+Create Date: 2024-11-12 09:49:53.558089
+
+"""
+
+from collections.abc import Sequence
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "69243fd24222"
+down_revision: str | None = "22298f3aac77"
+branch_labels: str | Sequence[str] | None = None
+depends_on: str | Sequence[str] | None = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
As before: I'd remove all these auto generated comments (as I assume you have reviewed the code and the comments is only a warning to do so)